torch_frame/datasets/multimodal_text_benchmark.py

```diff
@@ -20,6 +20,8 @@ class MultimodalTextBenchmark(torch_frame.data.Dataset):
         name (str): The name of the dataset to download.
         text_stype (torch_frame.stype): Text stype to use for text columns
             in the dataset. (default: :obj:`torch_frame.text_embedded`)
+        num_rows (int, optional): Number of rows to subsample.
+            (default: :obj:`None`)
```

Comment:
Why?

Reply:
The full dataset was too large and take hours to run. But since we are moving to using a synthetic dataset we are all good.